src/luarocks/cmd/upload.lua

```diff
@@ -14,6 +14,8 @@ function upload.add_to_parser(parser)
       :summary("Upload a rockspec to the public rocks repository.")
 
    cmd:argument("rockspec", "Rockspec for the rock to upload.")
+   cmd:argument("src_rock", "A corresponding .src.rock file; if not given it will be generated.")
```

Comment:
generally we use kebab-case for arguments
```suggestion
   cmd:argument("src-rock", "A corresponding .src.rock file; if not given it will be generated.")
```